design-system/uilib/src/components/dpdhl-radio-group/dpdhl-radio-group.stories.svelte

```diff
@@ -0,0 +1,44 @@
+
+<script lang="ts">
+    import { Meta, Template, Story } from '@storybook/addon-svelte-csf'
+    import { DPDHLRadioGroup, DPDHLRadioItem } from './index'
```

Comment:
we should not use the svelte components directly but the web-components from `@sprinteins/dpdhl-uilib`

eg.: instead of `DPDHLRadioGroup` use `dpdhl-radio-group`